src/Jackett.Common/Definitions/hdcity.yml

```diff
@@ -144,6 +144,8 @@
     keywordsfilters:
       - name: re_replace
         args: ["S0?(\\d{1,2})E(\\d{1,2})", "$1x$2"]
+      - name: re_replace
```

Comment:
try using the diacritics function instead
https://github.com/Jackett/Jackett/wiki/Definition-format#diacritics

Reply:
mind you,  diacritic replacement wont work for when the title on hdcity does actually have one.
I take it that if you search for `Caligula` on hdcity directly from your browser, you will not find `El Sangriento Imperio Romano Calígula`  ?

Reply:
Exactly what you say, searching for `Caligula` it doesn´t show `El Sangriento Imperio Romano Calígula` and that´s why the diacritic won´t work. I need to be able to search both with and without diacritics, as the tracker doesn´t have a standardised format